sectorbuilder.go

```diff
@@ -444,31 +448,22 @@ func sizeFromConfig(cfg Config) (abi.SectorSize, error) {
 }
 
 func sizeFromProofType(p abi.RegisteredProof) (abi.SectorSize, error) {
```

Comment:
likely want to just use https://github.com/filecoin-project/specs-actors/pull/212/files once merged